docs/CHANGELOG.asciidoc

```diff
@@ -131,6 +131,15 @@ Observability::
 
 Review the following information about the 7.17.1 release.
 
+[float]
+[[known-issue-v7.17.1]]
+=== Known issue
+
+Lens & visualizations::
+A change in the Markdown library that *TSVB* *Markdown* visualizations and *Text* dashboard panels use renders some tables differently.
```

Comment:
I think we should link to this change: https://github.com/markdown-it/markdown-it/pull/767